vividus-plugin-mobile-app/src/main/resources/properties/profile/mobile_app/profile.properties

```diff
@@ -2,3 +2,5 @@ spring.profiles.active=mobile_app
 
 mobile-app=true
 mobile-application.session-scope=story
+mobile-application.swipe.stabilization-duration=PT1S
```

Comment:
isn't it to much?

Reply:
it's tested and seems enough to wait for scroll disappearance after we stop swiping

Reply:
discussed to keep pt1s